src/app/portlets/dot-edit-page/content/components/dot-edit-page-workflows-actions/dot-edit-page-workflows-actions.component.ts

```diff
@@ -0,0 +1,32 @@
+import { Component, OnInit, Input } from '@angular/core';
+import { MenuItem } from 'primeng/primeng';
+import { DotWorkflowActions } from '../../../../../shared/models/dot-workflow-actions/dot-workflow-actions.model';
+import { WorkflowService } from '../../../../../api/services/workflow/workflow.service';
+import { Observable } from 'rxjs/Observable';
+
+@Component({
+    selector: 'dot-edit-page-workflows-actions',
```

Comment:
Please add the empty scss file ```dot-edit-page-workflows-actions.component.scss``` and referece it in the component, in the future we might need it.